initfiles/componentfiles/configxml/cgencomplist_win.xml

```diff
@@ -70,6 +70,9 @@
   <Component name="backupnode" processName="BackupNode" schema="backupnode.xsd">
     <File name="backupnode.xsl" method="xslt" destName="backupnode.xml"/>
   </Component>
+  <Component name="sparkthor" processName="SparkThor" schema="sparkThor.xsd">
+    <File name="sparkThor.xsl" method="xslt" destName="spark-env.conf"/>
```

Comment:
this entry differs between cgencomplist_win.xml and cgencomplist_linux.xml by the desName:
"spark-env.conf" vs "spark-env.sh" is this intentional?